pipeline/outputs/kinesis.md

```diff
@@ -19,7 +19,7 @@ See [here](https://github.com/fluent/fluent-bit-docs/tree/43c4fe134611da471e706b
 | region | The AWS region. |
 | stream | The name of the Kinesis Streams Delivery stream that you want log records sent to. |
 | time\_key | Add the timestamp to the record under this key. By default the timestamp from Fluent Bit will not be added to records sent to Kinesis. |
-| time\_key\_format | strftime compliant format string for the timestamp; for example, the default is '%Y-%m-%dT%H:%M:%S'. This option is used with time\_key. |
+| time\_key\_format | strftime compliant format string for the timestamp; for example, the default is '%Y-%m-%dT%H:%M:%S'.Support millisecond with '%3N' and support nanosecond with '%9N' and '%L'; for example, adding '%3N' to support millisecond '%Y-%m-%dT%H:%M:%S.%3N'. This option is used with time\_key. |
```

Comment:
grammar/language nit: I believe it reads better as "Supports millisecond precision '%3N' and supports nanosecond precision with..."